offchain/create_invoice.js

```diff
@@ -112,6 +114,16 @@ module.exports = (args, cbk) => {
         cbk);
       }],
 
+      // Calculate invoice expiry
+      expiresAt: ['validate', ({}, cbk) => {
+        // Exit early if expiry is not defined
+        if (!args.expires_in) {
+          return cbk();
+        }
+
+        return cbk(null, moment().add(args.expires_in, 'hour').toISOString());
```

Comment:
I think you could just do `new Date(Date.now() + (msPerSecond * secondPerMinute * hours)).toISOString()`, would that be enough?